ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -692,6 +692,12 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		deleted := struct{}{}
 		numDeletedUnhealthyWorkerPods := 0
 		for _, workerPod := range workerPods.Items {
+			logger.Info("reconcilePods", "Worker Pod Name: ", workerPod.Name, " Status: ", workerPod.Status.Phase)
```

Comment:
With this change we're now logging "reconcilePods" with the worker pod name multiple times (line 695, 697 and 702), is that expected?

Reply:
Hey @andrewsykim , this message `reconcilePods` indicates that these logs come from the reconcilePods function. However, I can change it if it's confusing or if you prefer a different format, wdyt?